<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>dspCreditCardTransactions</class>
 <widget class="QWidget" name="dspCreditCardTransactions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credit Card Transactions</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_6">
     <property name="margin">
      <number>12</number>
     </property>
     <property name="spacing">
      <number>12</number>
     </property>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="CustomerSelector" name="_customerSelector"/>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="_close">
           <property name="text">
            <string>&amp;Close</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_query">
           <property name="text">
            <string>&amp;Query</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="_showGroup">
       <property name="title">
        <string>Show</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QRadioButton" name="_alltrans">
            <property name="text">
             <string>All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="_processed">
            <property name="text">
             <string>Processed</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="_pending">
            <property name="text">
             <string>Pending</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="XCheckBox" name="_expired">
            <property name="text">
             <string>Expired</string>
            </property>
            <property name="forgetful">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>5</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="_validDaysLit">
            <property name="text">
             <string>Valid Days:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="_validDays">
            <property name="maximum">
             <number>30</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QFrame" name="_frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="XTreeWidget" name="_preauth">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QVBoxLayout">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="_postPreauth">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Post</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="_voidPreauth">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Void</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="_CCAmountLit">
              <property name="text">
               <string>Amount:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="CurrCluster" name="_CCAmount">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="localControl">
               <bool>false</bool>
              </property>
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="5" margin="5"/>
 <customwidgets>
  <customwidget>
   <class>CurrCluster</class>
   <extends>CurrDisplay</extends>
   <header>currcluster.h</header>
  </customwidget>
  <customwidget>
   <class>CurrDisplay</class>
   <extends>QWidget</extends>
   <header>currcluster.h</header>
  </customwidget>
  <customwidget>
   <class>CustomerSelector</class>
   <extends>QWidget</extends>
   <header>customerselector.h</header>
  </customwidget>
  <customwidget>
   <class>XCheckBox</class>
   <extends>QCheckBox</extends>
   <header>xcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>XTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>xtreewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_preauth</tabstop>
  <tabstop>_postPreauth</tabstop>
  <tabstop>_voidPreauth</tabstop>
  <tabstop>_CCAmount</tabstop>
  <tabstop>_close</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_close</sender>
   <signal>clicked()</signal>
   <receiver>dspCreditCardTransactions</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>717</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_processed</sender>
   <signal>toggled(bool)</signal>
   <receiver>_expired</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_processed</sender>
   <signal>toggled(bool)</signal>
   <receiver>_validDays</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>578</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
